bindings/rust/evmc-vm/src/lib.rs

```diff
@@ -320,7 +369,7 @@ impl From<ffi::evmc_result> for ExecutionResult {
     }
 }
 
-fn allocate_output_data(output: Option<Vec<u8>>) -> (*const u8, usize) {
+fn allocate_output_data(output: Option<&Vec<u8>>) -> (*const u8, usize) {
```

Comment:
Made this improvement here because first tried to use this in `call()`, but realised there's no need to duplicate memory.